src/CoreWlan/CWKeychain.cs

```diff
@@ -0,0 +1,231 @@
+using System;
+using System.Runtime.InteropServices;
+
+using CoreFoundation;
+using Foundation;
+using ObjCRuntime;
+using Security;
+
+#if NET
+using System.Runtime.Versioning;
+#endif
+
+
+#nullable enable
+
+using OSStatus = System.Int32;
+using SecIdentityRef = System.IntPtr;
+using CFArrayRef = System.IntPtr;
+using NSDataRef = System.IntPtr;
+using NSStringRef = System.IntPtr;
+
+namespace CoreWlan {
+#if NET
+	[SupportedOSPlatform ("maccatalyst15.0")]
+#else
+	[MacCatalyst (15,0)]
+#endif
+	public static partial class CWKeychain {
+
+#if NET
+		[SupportedOSPlatform ("macos10.9")]
+#else
+		[Mac (10,9)]
+#endif
+		[DllImport (Constants.CoreWlanLibrary)]
+		static extern OSStatus CWKeychainCopyWiFiEAPIdentity (CWKeychainDomain domain, NSDataRef ssid, out SecIdentityRef identity);
+
+#if NET
+		[SupportedOSPlatform ("macos10.9")]
+#else
+		[Mac (10,9)]
+#endif
+		public static bool FindWiFiEAPIdentity (CWKeychainDomain domain, NSData ssid, out SecIdentity? identity)
```

Comment:
This one does not follow the naming of the others:

```suggestion
		public static bool TryFindWiFiEAPIdentity (CWKeychainDomain domain, NSData ssid, out SecIdentity? identity)
```